en/publish/publish-bytedance.md

```diff
@@ -0,0 +1,124 @@
+
+For details about ByteDance  access, please refer to the [ByteDance Mini Game Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/registration).
+
+## Preparation
+
+- Download the [ByteDance Developer Tools](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/developer-instrument-update-and-download) and install it.
+
+- Refer to the [ByteDance Mini Game Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/registration) to register, login and apply for mini games on the ByteDance Developer Platform.
+
+- After the application is approved, you can find the appid in **Development Management -> Development Settings** of the developer platform.
+
+  ![](./publish-bytedancegame/appid.png)
+
+## Publishing Process
+
+Use Cocos Creator to open the project to be published, open the **Build...** panel from **Menu Bar -> Project**, select **ByteDance Mini Game** in **Platform**, fill in the appid, and click **Build**.
+
+![](./publish-bytedancegame/build.png)
+
+### Parameter Configuration
+
+The specific rules for filling in the relevant parameter configuration are as follows.
+
+- **The Compression Type of the Main Package**
+
+  To set the compression type of the main package, please refer to the [Asset Bundle - Compression Type](../asset-manager/bundle.md#compression-type) documentation.
+
+- **Configure the Main Package as a Remote Package**
+
+  This is optional and needs to be used in conjunction with the **Resource Server Address** option. <br>
+  When checked, the main package is configured as a remote package and is built with its associated dependencies to the built-in Asset Bundle -– [main](../asset-manager/bundle.md#the-built-in-asset-bundle) in the remote directory of the release package. The developer needs to upload the entire remote folder to the remote server.
+
+- **Initial Scene Subpackage**
+
+  This item is optional. <br>
+  When checked, the first scene and its related dependencies will be built into the built-in Asset Bundle under the release package directory assets –– [start-scene](../asset-manager/bundle.md#the-built-in-asset-bundle) to improve the speed of loading assets for the initial scene. The details can be found in the document [Asset Loading for Initial Scenes](publish-wechatgame.md#speed-up-the-loading-of-the-start-scene).
+
+- **Resource Server Address**
+
+  This item is optional and is used to fill in the address of the remote server where the resource is stored. Developers need to manually upload the remote folder in the release package directory to the filled resource server address after the build.
+
+- **Code Directory for Open Data Domains**
+
+  This item is optional and is used to access the open data domain of ByteDance , please refer to [Access the Open Data Domain of ByteDance Mini Game](publish-bytedance-sub-domain.md) for the specific publishing process.
```

Comment:
This item is optional and is used to access the open data domain of ByteDance , please refer to [Access the Open Data Domain of ByteDance Mini Game](publish-bytedance-sub-domain.md) for the specific publishing process.
-->
This item is optional and is used to access the open data domain for ByteDance. Please refer to the [Access the Open Data Domain of ByteDance Mini Game](publish-bytedance-sub-domain.md) documentation for the specific publishing process.